posts/2022-01-31-multi-party-data-sharing-with-control.md

```diff
@@ -0,0 +1,271 @@
+
+A core tenet of Vendia Share is _sharing with control_.  That tenet is the basis for a number of features that allow participants to gain trust in the Vendia Share platform, and with each other.  Those features, among many others, make Vendia Share the best platform to consider when building multi-party, real-time data sharing solutions. It's these same features that bring the cost and complexity of building real-time data sharing solutions down significantly.  Because sharing with control is a core tenet, it's not surprising that access controls have been built into multiple levels of the Vendia Share architecture.
+
+In this blog post we'll explore a range of Vendia Share controls, including their purpose and placement within the overall Vendia Share architecture, and their relationship with each other.
+
+* **[User Management Access Controls](https://www.vendia.net/docs/share/rbac#user-actions)** - Protect Vendia Share user account management operations
+* **[Uni Management Access Controls](https://www.vendia.net/docs/share/rbac#uni-actions)** - Protect Vendia Share Uni management operations
+* **[Node Access Controls](https://www.vendia.net/docs/share/node-access-control)** - Protects a node's GraphQL API
+* **[Data Access Controls](https://www.vendia.net/docs/share/fine-grained-data-permissions)** - Protects the data stored in a Uni
+
+<figure>
+    <img src="https://user-images.githubusercontent.com/85032783/151488928-76a5b185-329e-424a-af46-17a397868716.png" />
+    <figcaption ><b>Figure 1</b> - <i>Vendia Share Access Controls</i></figcaption>
+</figure>
+
+As captured in Figure 1, the user and Uni access controls protect the Vendia Share control plane and the node and data access controls protect the Vendia Share data plane.  You can differentiate control and data plane access controls by considering whether the access control protects the Vendia Share platform (if so, it's a _control plane_ protection) or a Uni or node (if so, it's a _data plane_ protection).  The most important point is that multiple access controls exist in each plane of the Vendia Share architecture.  In addition to providing defense-in-depth, the numerous available access controls offer each participant in a Uni the flexibility and control they often need to willingly share data with others.
```

Comment:
May want to preface this by describing the architectural boundary between control plane and data plane. I think some readers may misinterpret "control protects the Vendia Share platform".

The way I think of it/explain it, is the control plane is for managing unis and nodes (metadata), and the data plane is for interacting with the data in the uni.

Reply:
Is this the first time we've talked about control plane and data plane publicly? Not sure if we want to consider other terms for public consumption

Reply:
Got the 👍 to use the terms publicly.  Will add an intro sentence or two to define the planes.